src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/InfosetObjects/XamlNodes.cs

```diff
@@ -115,7 +115,7 @@ public XamlNode(LineInfo lineInfo)
 
         public override string ToString()
         {
-            string str = String.Format(TypeConverterHelper.InvariantEnglishUS, "{0}: ", NodeType);
+            string str = NodeType.ToString(TypeConverterHelper.InvariantEnglishUS);
```

Comment:
This interpolation is different from what was present earlier.

Reply:
Why not use `string.Create` here instead ?

Reply:
You're right, I've just fixed it